w3_d4_2_js_integration_testing/README.md

```diff
@@ -0,0 +1,85 @@
+# API Testing with Javascript
+
+| Objectives |
+| :--- |
+| Add basic test coverage to any app through integration tests |
+| Use Test Driven Development and Behavior Driven Development |
+
+| Concepts | Tools | Activities |
+| :---: | :---: | :---: |
+| Integration Testing TDD/BDD | Mocha, Supertest, Chai | Challenges |
+
+### Preparatory Readings
+
+1. [Difference Between Test Methodologies](http://stackoverflow.com/questions/4904096/whats-the-difference-between-unit-functional-acceptance-and-integration-test/4904533#4904533)
+
+
+### Motivation (Why?)
+
+### Analogy (What?)
+
+### Key Concepts & Snippets
+
+#### Unit & Integration Tests
+
+**Unit Tests** test only one part of the code independently.
+
+**Integration Tests** test all the code the way a user would use it.
+
+#### Example of a Simple API Test
+
```

Comment:
Can you please add comments to this to explain what each part of the code is doing?